angularjs-portal-frame/src/main/webapp/main.js

```diff
@@ -0,0 +1,59 @@
+require.config({
+
+  paths: {
+    'my-app': 'my-app.js'
+    /* TODO: Incorporate the <scripts/> that are being globally imported in frame.html

```

Comment:
Happy to - and I started to - but I was biting off a bit too much at once (the bower config could also use TLC), so I thought it was better to start with the minimum functional change and circle back to get things better organized.